randovania/games/samus_returns/logic_database/Area 1.txt

```diff
@@ -1029,41 +1029,43 @@ Extra - asset_id: collision_camera_033
   * Extra - actor_name: LE_Item_001
   * Extra - actor_type: item_missiletank
   > Door to Moheek Mount
-      Trivial
+      All of the following:
+          Varia Suit
+          Gravity Suit or Space Jump or Use Spider Ball
 
 > Door to Moheek Mount; Heals? False
   * Layers: default
   * Charge Beam Door to Moheek Mount/Door to Magma Pool
   * Extra - actor_name: Door009
   * Extra - actor_type: doorchargecharge
   > Pickup (Missile Tank)
-      Any of the following:
-          All of the following:
-              Varia Suit
+      All of the following:
+          Varia Suit
+          Any of the following:
               Space Jump or Use Spider Ball
-          All of the following:
-              # Gravity way
-              Gravity Suit
-              Wall Jump (Beginner) or Simple IBJ
-              # Super Jump + Unmorph Extend
-              Morph Ball and Super Jump (Advanced) and Unmorph Extend (Advanced)
+              All of the following:
+                  Gravity Suit
```

Comment:
i would add comments here that indicate that these are dont-touch-lava and touch-lava approaches